assemblies/pentaho-cde-renderer/docker/pom.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.pentaho.ctools</groupId>
+    <artifactId>pentaho-cde-renderer-assembly</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-cde-renderer-docker</artifactId>
```

Comment:
Drop the `pentaho` prefix in the artifactId. We need it for the feature artifact but I see no reason to use it here. `org.pentaho.ctools/cde-renderer-docker` seems clear enough.